auto_submit/lib/service/validation_service.dart

```diff
@@ -128,40 +145,154 @@ class ValidationService {
     for (ValidationResult result in results) {
       if (!result.result && result.action == Action.REMOVE_LABEL) {
         final String commmentMessage = result.message.isEmpty ? 'Validations Fail.' : result.message;
-        await gitHubService.createComment(slug, prNumber, commmentMessage);
-        await gitHubService.removeLabel(slug, prNumber, Config.kAutosubmitLabel);
-        log.info('auto label is removed for ${slug.fullName}, pr: $prNumber, due to $commmentMessage');
+
+        String message = 'auto label is removed for ${slug.fullName}, pr: $prNumber, due to $commmentMessage';
```

Comment:
Another lint we should add:

https://dart-lang.github.io/linter/lints/prefer_final_locals.html

Reply:
With these linter changes I will make a separate pr for these. But I will make changes in this file for the local final variables.

Reply:
Any local that can be made final in this file has been made final.